lib/kb_Msuite/Utils/DataStagingUtils.py

```diff
@@ -55,22 +55,47 @@ def stage_input(self, input_ref, fasta_file_extension):
         #     the object.
         obj_name = input_info[1]
         type_name = input_info[2].split('-')[0]
+
+        # Standard Single Assembly
         if type_name in ['KBaseGenomeAnnotations.Assembly', 'KBaseGenomes.ContigSet']:
             au = AssemblyUtil(self.callback_url)
             os.makedirs(input_dir)
             filename = os.path.join(input_dir, obj_name + '.' + fasta_file_extension)
             au.get_assembly_as_fasta({'ref': input_ref, 'filename': filename})
             if not os.path.isfile(filename):
                 raise ValueError('Error generating fasta file from an Assembly or ContigSet with AssemblyUtil')
+            # make sure fasta file isn't empty
+            if self.count_fasta_seq_len(filename) == 0:
+                raise ValueError('Assembly or ContigSet is empty in filename: '+str(filename))
```

Comment:
I don't see any test for this logic

Reply:
Fine.  I'll add it.  It's kinda hard to mess up code this trivial.